### ISSUE TITLE
chore(release): publish v1.1.0

### ISSUE THREAD

**Issue by petermetz** [hyperledger/cactus#2054]
### Description

We need to issue the 1.1.0 release because it will unblock a few security fixes and other release management type blockers that we have (for example the dependency chain between npm package releases and container images)

The main reason I create a separate issue for the release issuance itself is that other issues can be marked as dependent on this one which makes some administrative chores we have a little more convenient and better defined.

cc: @aldousalvarez
